mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -178,6 +182,8 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "update_version_stat_collection" => handle_mmrpc(ctx, request, update_version_stat_collection).await,
         "verify_message" => handle_mmrpc(ctx, request, verify_message).await,
         "withdraw" => handle_mmrpc(ctx, request, withdraw).await,
+        "withdraw_erc1155" => handle_mmrpc(ctx, request, withdraw_erc1155).await,
+        "withdraw_erc721" => handle_mmrpc(ctx, request, withdraw_erc721).await,
```

Comment:
This should be unified like `withdraw_erc_tokens`

Reply:
what about `withdraw_nft` name? 
well yeah, currently we are adding  just EVM support. however if we have to add not EVM based protocol in the future, we will need to create `withdraw_new_nft_standard_name` method. 
As we have plans to simplify API interface, lets leave `withdraw_nft` name?

Reply:
`withdraw_nft` seems also fine